current/local-cli/rnpm/windows/src/windows.js

```diff
@@ -8,7 +8,8 @@ const Common = require('./common');
 const chalk = require('chalk');
 const execSync = require('child_process').execSync;
 const path = require('path');
-const prompt = require('@react-native-community/cli/build/tools/generator/promptSync').default();
```

Comment:
local-cli\generator-common\index.js also require's and uses this module.  Does that need to be changed as well?  prompts looks better anyway.

Reply:
Thats in react-native-windows which declares the cli dep already.

Reply:
That dependency is about to be removed when we move to 61

Reply:
If its removed the generator will have to be updated to not use it.

I expect that over time we will require the cli dependency as we work to properly implement auto-linking etc -- so maybe we should just keep the dependency and update it to match the same version range that react-native itself uses in 61.